docs/getting-started/linux-installation.md

```diff
@@ -26,66 +26,37 @@ ExecStart=/usr/bin/dockerd -H fd:// --dns=172.17.0.1
 ```
 1. Set up the docker0 network as trusted
     - `sudo firewall-cmd --zone=trusted --add-interface=docker0 && sudo firewall-cmd --zone=trusted --add-interface=docker0 --permanent`
-1. Start the docker daemon
-    - `sudo systemctl start docker`
+1. Restart the docker daemon
+    - `sudo systemctl restart docker`
 
 ## Linux installation on Ubuntu/Debian
 
 1. [Install Docker for Ubuntu](https://docs.docker.com/engine/installation/linux/ubuntu/) or 
 [Install Docker for Debian](https://docs.docker.com/engine/installation/linux/debian/)
 1. [Install Docker Compose](https://docs.docker.com/compose/install/)
-
-**If using Upstart**
-
-- Set the DNS configuration for dnsdock, as well as known RFC-1918 address space
-    - Please note that the following command will over-write your existing Docker daemon configuration file.  Please 
-    set the -dns=172.17.0.1 option manually as an alternative
-    - `echo 'DOCKER_OPTS="-dns=172.17.0.1"' | sudo tee /etc/default/docker`
-- Start the docker daemon
-    - `sudo start docker`
-
-**If using Systemd**
-
-- Set the DNS configuration for Docker
+1. Set the DNS configuration for Docker
     - We need to modify the command that docker uses within systemd
     - `sudo mkdir /etc/systemd/system/docker.service.d`
     - `sudo vi /etc/systemd/system/docker.service.d/docker.conf`
-    - In that file put something like the following:    
-    
+    - In that file put something like the following:      
 ```bash
 [Service]
 ExecStart=
 ExecStart=/usr/bin/dockerd -H fd:// --dns=172.17.0.1
 ```
+1. Restart the docker daemon
```

Comment:
Worth double-checking, but I believe Markdown doesn't care what number you used. For source readability maybe we should increment?

Reply:
Agree